synthtool/gcp/templates/node_split_library/index.ts.j2

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
```

Comment:
I don't think we want the `.j2` extension? I noticed the upstream generated template ends up having:

```
index.ts.jinja2 
```

☝️ as the name.

Reply:
The reason I use `.j2` instead of `jinja2` is the rest of repo use `.j2`.

upstream generated template `index.ts.jinja2` issue resolve. It should not copy a `index.ts.jinja2` when synthtool is called. Update the upstream [PR](https://github.com/googleapis/nodejs-text-to-speech/pull/400).